src/components/Compare/PopulationsTable.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslation } from "react-i18next";
+
+import { CareProvider, POPULATIONS_SERVED } from "../../types";
+import CompareStripedRows from "./CompareStripedRows";
+
+function PopulationsTable({
+  providerA,
+  providerB,
+}: {
+  providerA: CareProvider;
+  providerB: CareProvider;
+}) {
+  const { t } = useTranslation();
+
+  const rowData = POPULATIONS_SERVED.map((pop) => {
+    return {
+      label: t(pop),
```

Comment:
noticed that the population keys are inconsistent w/ the other translation keys b/c they don't have a "populationValues" prefix. but didn't bother fixing b/c it's extremely nbd, just flagging in case y'all do any copy spreadsheet cleanup down the line

Reply:
Might be worthwhile to note this in the code!